### PR TITLE
reg table ux update

### PR DESCRIPTION
**Pull request by kialj876**
Signed-off-by: Kial Jinnah <kialj876@gmail.com>

*Issue #:* /bcgov/entity#11114

*Description of changes:*
- reg table info added to store to support all flows (can be used later for other things)
- adding a reg/draft will scroll to new reg with ux stuff
- tests updated

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/store/state/state-model.ts [hunk 1]
```diff
@@ -18,6 +18,29 @@ export const stateModel: StateModelIF = {
     certified: false,
     legalName: ''
   },
+  folioOrReferenceNumber: '',
```
Comment (kialj876) [inline ppr-ui/src/store/state/state-model.ts:21]: just moved this here so its alphabetical to make the store a little easier to navigate now that we have quite a few sections

File: ppr-ui/src/store/state/state-model.ts [hunk 2]
```diff
@@ -18,6 +18,29 @@ export const stateModel: StateModelIF = {
...
+  // orig reg party used for discharge/renew/amend, other for amend only
+  originalRegistration: {
```
Comment (kialj876) [inline ppr-ui/src/store/state/state-model.ts:23]: same thing here

File: ppr-ui/src/store/state/state-model.ts [hunk 3]
```diff
@@ -57,27 +80,9 @@ export const stateModel: StateModelIF = {
...
-      lienAmount: '',
-      surrenderDate: ''
-    },
-    parties: {
-      valid: true,
-      registeringParty: null,
-      securedParties: [],
-      debtors: []
-    }
+  registrationTable: {
```
Comment (kialj876) [inline ppr-ui/src/store/state/state-model.ts:83]: this is the only new part of the store I added

File: ppr-ui/src/store/state/state-model.ts [hunk 4]
```diff
@@ -89,6 +94,7 @@ export const stateModel: StateModelIF = {
     searching: false,
     searchCertified: false
   },
+  staffPayment: null,
```
Comment (kialj876) [inline ppr-ui/src/store/state/state-model.ts:97]: same thing here as above (alphabetical move)

**Review by jordiwes — APPROVED** [review]
Phew! Good stuff!

**Review by doug-lovett — APPROVED** [review]
Looks good.

**Pull request merged**